test/pylib/minio_server.py

```diff
@@ -203,13 +208,16 @@ def _get_environs(self):
 
     def _unset_environ(self):
         for env in self._get_environs():
-            del os.environ[env]
+            if self.old_env[env] is not None:
+                os.environ[env] = self.old_env[env]
+            else:
+                del os.environ[env]
 
     def print_environ(self):
         msgs = []
         for key in self._get_environs():
             value = os.environ[key]
-            msgs.append(f'export {key}={value}')
+            msgs.append(f'APA export {key}={value}')
```

Comment:
What does APA stand for here?

Reply:
Oops. Sorry. Lets a debug change there. Just helped me distinguish things when figuring out who was messing with the env in test.py hierarchy. Removed.